src/core/rr.rkt

```diff
@@ -1,40 +1,11 @@
 #lang racket
 
 (require "../utils/common.rkt"
-         "programs.rkt"
          "../utils/timeline.rkt"
-         "rules.rkt"
-         "egg-herbie.rkt"
-         "matcher.rkt")
+         "egg-herbie.rkt")
 
 (provide rewrite-expressions)
 
-(define (rule-apply rule expr)
-  (let ([bindings (pattern-match (rule-input rule) expr)])
-    (if bindings (cons (pattern-substitute (rule-output rule) bindings) bindings) #f)))
-
-;;
-;;  Non-recursive rewriter
-;;
-
-;; Applies rules verbatim
-(define (rewrite-once expr ctx #:rules rules)
-  ;; actually match
-  (define rule-apps (make-hash))
-  (define expr-repr (repr-of expr ctx))
-  (define changelists
-    (reap [sow]
-          (for ([rule rules] #:when (equal? expr-repr (rule-otype rule)))
-            (let* ([result (rule-apply rule expr)])
-              (when result
-                (hash-update! rule-apps (rule-name rule) add1 0)
-                (sow (list (car result) rule)))))))
-  ;; rule statistics
-  (for ([(name count) (in-hash rule-apps)])
-    (when (> count 0)
-      (timeline-push! 'rules (~a name) count)))
-  changelists)
-
```

Comment:
I guess it's no longer possible to turn off rr?

Reply:
Yeah that was an unused code path. I realized it when I was trying to update it.